rdr_service/genomic/genomic_job_controller.py

```diff
@@ -854,23 +856,38 @@ def load_raw_awn_data_from_filepath(self, file_path):
 
     def create_incident(self, **kwargs):
         """
-        Creates an
-        :return: GenomicIncident
+        Creates an GenomicIncident and sends alert via Slack if default
+        for slack kwarg is not overridden
+        :return:
         """
-        with self.incident_dao.session() as session:
-            return session.add(GenomicIncident(**kwargs))
+        insert_kwargs = {key: value for key, value in kwargs.items()
+                         if key in GenomicIncident.__table__.columns.keys()}
+        incident = self.incident_dao.insert(GenomicIncident(**insert_kwargs))
+
+        if kwargs.get('slack') is False:
+            return
```

Comment:
I don't think the default behavior should be to send an alert. We insert records to `genomic_incident` for every row that fails to ingest, so for every ingestion we would need to remember to add the kwarg for slack alerts.

Reply:
2350cec964f2226912ac6f75845502e0f1cb5ea6